robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/ParticipationWrapper.java

```diff
@@ -48,6 +48,11 @@ public BigDecimal getInterestRate() {
         return participation.getInterestRate();
     }
 
+    @Override
+    public BigDecimal getRevenueRate() { // TODO check if Participation has this field too
```

Comment:
Did you manage to find out more to resolve this TODO?

Reply:
I completely missed this one. However, it doesn't really matter, @jhrcek - since the data point is loaded from the `Loan`. The switch to `Participation` would be a perf optimization, but nothing would change otherwise - this rule already works, as is, in all the cases.